stl/inc/yvals_core.h

```diff
@@ -933,6 +933,17 @@
 #define _CXX20_DEPRECATE_ATOMIC_INIT
 #endif // ^^^ warning disabled ^^^
 
+#if _HAS_CXX20 && !defined(_SILENCE_CXX20_VOLATILE_DEPRECATION_WARNING) \
+    && !defined(_SILENCE_ALL_CXX20_DEPRECATION_WARNINGS)
+#define _CXX20_DEPRECATE_VOLATILE                                                \
+    [[deprecated("warning STL4029: "                                             \
+                 "The volatile keyword has been partially deprecated in C++20. " \

```

Comment:
The other deprecation messages use present tense, so I suggest:

```suggestion
                 "The volatile keyword is partially deprecated in C++20. " \
```